hacking/build_library/build_ansible/filters.py

```diff
@@ -0,0 +1,99 @@
+# Copyright: (c) 2019, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+import re
+from jinja2.runtime import Undefined
+
+try:
+    from html import escape as html_escape
+except ImportError:
+    # Python-3.2 or later
+    import cgi
+
+    def html_escape(text, quote=True):
+        return cgi.escape(text, quote)
```

Comment:
Minor: jinja2 import should come after the html imprt (html is from the stdlib.  jinja2 is third-party)

Reply:
Are you saying `from jinja2.runtime import Undefined` should come after this import statement?

Reply:
Correct